### PR TITLE
[onert] Introduce I/O size getter for package

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit introduces I/O size getter in nnfw_api_internal.cc and NNPkg.h to reduce code duplication. 
New getters support multimodel and pipeline.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
acl-cl backend: 
```
15:42:44  not ok 1 - MODELS/inception_module
```

**Comment by hseok-oh** [comment]
@nnfw-bot test cross-runtime-test-debug

File: runtime/onert/api/src/nnfw_api_internal.cc [hunk 1]
```diff
@@ -1127,6 +1127,34 @@ const onert::ir::Graph *nnfw_session::primary_subgraph()
...
+{
+  if (isStateInitialized())
+    throw std::runtime_error{"Model is not loaded yet"};
+
+  if (isStateModelLoaded())
+    return _nnpkg->outputSize();
+
+  if (_executions.empty())
+    return _compiler_artifact->_executors->outputSize();
+
```
Comment (chunseoklee) [inline runtime/onert/api/src/nnfw_api_internal.cc:1154]: (optional) may need comment about pipeline ?

**Review by chunseoklee — DISMISSED** [review]

File: runtime/onert/api/src/nnfw_api_internal.cc [hunk 2]
```diff
@@ -1127,6 +1127,34 @@ const onert::ir::Graph *nnfw_session::primary_subgraph()
   }
 }
 
+uint32_t nnfw_session::getInputSize()
+{
+  if (isStateInitialized())
+    throw std::runtime_error{"Model is not loaded yet"};
+
+  if (isStateModelLoaded())
```
Comment (ragmani) [inline runtime/onert/api/src/nnfw_api_internal.cc:1135]: Is there any case called when model is not loaded? So how about changing to error handling?

Comment (hseok-oh) [reply runtime/onert/api/src/nnfw_api_internal.cc:1135]: You can find error handling on Line 1132~1133.

**Review by chunseoklee — APPROVED** [review]

File: runtime/onert/api/src/nnfw_api_internal.cc (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/api/src/nnfw_api_internal.cc:1157]: I've added comment.

**Review by glistening — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]

**Pull request merged**